apis/projectcontour/v1/httpproxy.go

```diff
@@ -929,6 +929,12 @@ type DownstreamValidation struct {
 	// +kubebuilder:validation:MinLength=1
 	CACertificate string `json:"caSecret,omitempty"`
 
+	// Name of a Kubernetes secret that contains a pem encoded crl.
+	// This will be used to verify that a client certificate has not been revoked
+	// +optional
+	// +kubebuilder:validation:MinLength=1
+	CACertificateRevocationList string `json:"caCrlSecret,omitempty"`
```

Comment:
In light of the awkward acronyms, maybe just `crlSecret` (as you have in the design doc) would be better for the YAML, no need to include `ca` as a prefix.

Reply:
Yep, forgot to update this